### ISSUE TITLE
Deprecation warning due to invalid escape sequences in Python 3.7

### ISSUE THREAD

**Issue by tirkarthi** [Boudewijn26/gTTS-token#19]
Deprecation warnings are raised due to invalid escape sequences. This can be fixed by using raw strings or escaping the literals.

```
find . -iname '*.py' | grep -v example | xargs -P 4 -I{} python3.8 -Wall -m py_compile {}  
./gtts_token/gtts_token.py:65: DeprecationWarning: invalid escape sequence \d
  result = re.search("\d{6}\.[0-9]+", tkk_expr).group(0)
./gtts_token/gtts_token.py:70: DeprecationWarning: invalid escape sequence \d
  a = re.search("a\\\\x3d(-?\d+);", tkk_expr).group(1)
./gtts_token/gtts_token.py:71: DeprecationWarning: invalid escape sequence \d
  b = re.search("b\\\\x3d(-?\d+);", tkk_expr).group(1)
```

**Comment by Boudewijn26** [comment]
Hi @tirkarthi, your title mentions Python 3.7 while your snippet shows Python 3.8. Would you be so kind to clarify which one it is exactly?

**Comment by tirkarthi** [comment]
This happens

**Comment by tirkarthi** [comment]
This happens in Python 3.7 too. Sorry closed by mistake.

**Comment by Boudewijn26** [comment]
This has been resolved in v1.1.4. Thank you for reporting
